### PR TITLE
🛠 Rework a lot of strings

### PR DESCRIPTION
**Pull request by Timeraa**
I reworked a lot of strings, fixed capitalizaion, fixed grammar issues and in general outdated stuff.

**Please go through those strings, tell me suggestions.**

### CODE REVIEW

File: src/Website/contributors.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
 		"description": "Tippy message when you hover on the avatar of a credit card."
 	},
 	"contributors.cards.userIdCopied": {
-		"message": "User ID copied to your clipboard!",
+		"message": "User Id copied!",
```
Comment (LewisTehMinerz) [inline src/Website/contributors.json:99]: Should be `ID` rather than `Id`

File: src/Website/downloads.json [hunk 2]
```diff
@@ -1,134 +1,138 @@
...
-  "downloads.button.done": {
-    "message": "Done",
-    "description": "Button on the page when trying to download with adblocker on."
-  },
+	"downloads.header.title": {
+		"message": "Time to show off.",
+		"description": "Title"
+	},
+	"downloads.header.subtitle": {
+		"message": "Use PreMiD now and show off to your friends what you're doing, maybe you find someone with the same interests.",
```
Comment (doomlerd) [inline src/Website/downloads.json:7]: "Use PreMiD now and share your favorite activities, maybe you could find someone with the same interests."

File: src/Website/downloads.json [hunk 3]
```diff
@@ -1,134 +1,138 @@
...
+	"downloads.instructions.step_2": {
+		"message": "Install our application",
+		"description": "Heading for installing the application"
+	},
+	"downloads.instructions.step_3": {
+		"message": "Add our Extension to your browser",
+		"description": "Heading for installing the application"
+	},
+	"downloads.instructions.step_4": {
+		"message": "Reload this site and check if the extension is connected",
```
Comment (doomlerd) [inline src/Website/downloads.json:27]: "Reload the page and check if the extension is connected"

File: src/Website/downloads.json [hunk 4]
```diff
@@ -1,134 +1,138 @@
...
+	"downloads.warning.differentVersion": {
+		"message": "Current version of this platform is {0} (this version is specific to Linux, and doesn't apply to other platforms).",
+		"description": "Tippy message of the Linux card."
+	},
+	"downloads.warning.differentVersion2": {
+		"message": "This version is specific to Linux, and doesn't apply to other systems. Application's main version is {0}",
+		"description": "Tippy message of the version number when user is on Linux."
+	},
+	"downloads.error.login": {
+		"message": "Please login in order to see these downloads.",
```
Comment (doomlerd) [inline src/Website/downloads.json:75]: "Please login in order to access downloads"

File: src/Website/downloads.json [hunk 5]
```diff
@@ -1,134 +1,138 @@
...
+		"description": "Error message if the user is not logged in."
+	},
+	"downloads.error.noaccess.title": {
+		"message": "Uh oh, it looks like you do not have alpha/beta access :(",
+		"description": "Error message if the user doesn't alpha/beta access."
+	},
+	"downloads.error.noaccess.description": {
+		"message": "You can join our beta program for free **here**. Hurry up! We only have {0} more slots available",
```
Comment (doomlerd) [inline src/Website/downloads.json:83]: "You can join our beta program **here** for free. Hurry up! we only have {0} more slots available"

File: src/Website/errors.json [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
+	"store.search.notFound": {
+		"message": "We couldn't find that Presence...",
+		"description": "Search result text when nothing's found"
+	},
+	"error.code.404": {
+		"message": "Sorry, we couldn't find that page.",
+		"description": "Error (404) message"
+	},
+	"error.code.4": {
+		"message": "You're looking for non existant or deleted Presence.",
```
Comment (doomlerd) [inline src/Website/errors.json:11]: "You're looking for a non existant or deleted Presence."

File: src/Website/errors.json [hunk 7]
```diff
@@ -1,46 +1,46 @@
...
+	"error.page.details": {
+		"message": "Details",
+		"description": "'Details' text on error page container"
+	},
+	"error.page.button": {
+		"message": "Back to homepage",
+		"description": "Button on the error page"
+	},
+	"user.notFound.heading": {
+		"message": "That user couldn't be found. Try to make sure that the",
```
Comment (doomlerd) [inline src/Website/errors.json:27]: remove the last "the", it's duped

File: src/Website/errors.json [hunk 8]
```diff
@@ -1,46 +1,46 @@
...
+	"user.notFound.message2": {
+		"message": "The user has created enough Presences and has the Presence Developer role in our Discord server.",
+		"description": "Check the presence amount of the developer created message."
+	},
+	"user.notFound.message3": {
+		"message": "Check if others are experiencing the same issue on our Discord server.",
+		"description": "Check if the error is also happening to others message."
+	},
+	"user.notFound.message4": {
+		"message": "If you believe this was a mistake, please contact us in [our Discord server].",
```
Comment (doomlerd) [inline src/Website/errors.json:43]: "If you believe this was a mistake, please contact us at [our Discord server]."

File: src/Website/header.json [hunk 9]
```diff
@@ -1,38 +1,38 @@
...
+		"message": "No, thanks",
+		"description": "Button that will decline notification."
+	},
+	"header.languageNotification.btnAccept": {
+		"message": "Switch language",
+		"description": "Button that switches language after click."
+	},
+	"header.loader.phrases": {
+		"message": "Baking pancakes...;Placing containers...;Parsing your IP address...;UwU;Breaking rules...;Wiping your hard drive...",
+		"description": "Randomly appears when website is loading data. [WARNING: Keep the `;` symbol, its used to separate phrases]"
```
Comment (doomlerd) [inline src/Website/header.json:24]: it's*

File: src/Website/home.json [hunk 10]
```diff
@@ -1,86 +1,86 @@
...
+	"home.features.heading": {
+		"message": "You will love these features!",
+		"description": "Features section"
+	},
+	"home.features.presencesystem.heading": {
+		"message": "Find **your** presence",
+		"description": "Features section subheading"
+	},
+	"home.features.presencesystem.description": {
+		"message": "We created our extension in a way that let's you add and use presences created by others and manage them in our extension!",
```
Comment (doomlerd) [inline src/Website/home.json:43]: "We created our extension in a way that lets you add, use and manage Presences created by our community with ease!"

File: src/Website/home.json [hunk 11]
```diff
@@ -1,86 +1,86 @@
...
+	"home.features.simpleInterface.description.1": {
+		"message": "App and Extension settings",
+		"description": "Features section description point"
+	},
+	"home.features.simpleInterface.description.2": {
+		"message": "Presence management",
+		"description": "Features section description point"
+	},
+	"home.features.simpleInterface.description.3": {
+		"message": "Sort by category",
```
Comment (doomlerd) [inline src/Website/home.json:67]: "Sorting by category" is better actually

File: src/Website/home.json [hunk 12]
```diff
@@ -1,86 +1,86 @@
...
+		"description": "Features section description point"
+	},
+	"home.features.quickSupport.heading": {
+		"message": "Quick support",
+		"description": "Features section subheading"
+	},
+	"home.features.quickSupport.description": {
+		"message": "Something is glitched, sending you tons of errors or simply not working? We will help you to get rid of those issues on our Discord server, we also created our very own support system that will make sure your issue get's solved as soon as possible.",
```
Comment (doomlerd) [inline src/Website/home.json:75]: "gets resolved"*

File: src/Website/jobs.json [hunk 13]
```diff
@@ -80,7 +80,7 @@
 		"description": "Benefit card title."
 	},
 	"jobs.benefit4.description": {
-		"message": "With your contribution it is possible to run this project.",
+		"message": "With your contribution its possible to run this project.",
```
Comment (doomlerd) [inline src/Website/jobs.json:83]: it's*

File: src/Website/jobs.json [hunk 14]
```diff
@@ -96,7 +96,7 @@
 		"description": "Benefit card title."
 	},
 	"jobs.benefit6.description": {
-		"message": "Be able to test new features before anyone else.",
+		"message": "Test new features before anyone else.",
```
Comment (doomlerd) [inline src/Website/jobs.json:99]: "Test new features earlier than anyone else."

File: src/Website/maintenance.json [hunk 15]
```diff
@@ -1,26 +1,26 @@
...
-  "maintenance.discordserver": {
-    "message": "Discord Server",
-    "description": "Discord Server link on maintenance page"
-  }
+	"maintenance.big": {
+		"message": "Oh noes!",
+		"description": "The big text on maintenance page"
+	},
+	"maintenance.heading": {
+		"message": "Unfortunately we are experiencing some technical difficulties. We will return shortly. Sorry for the inconvenience.",
```
Comment (doomlerd) [inline src/Website/maintenance.json:7]: "Unfortunately we are experiencing some technical difficulties. We will return shortly, sorry for the inconvenience."

File: src/Website/store.json [hunk 16]
```diff
@@ -1,102 +1,98 @@
...
+	"store.category.other": {
+		"message": "Other",
+		"description": "Category at the top"
+	},
+	"store.search": {
+		"message": "Search",
+		"description": "Placeholder for presence search"
+	},
+	"store.search.filters": {
+		"message": "Enter",
```
Comment (doomlerd) [inline src/Website/store.json:91]: maybe this could be "enter" without uppercase since it's repeated

**Pull request merged**